src/Tribe/Rewrite.php

```diff
@@ -10,6 +10,20 @@
  * Permalinks magic Happens over here!
  */
 class Tribe__Events__Rewrite extends Tribe__Rewrite {
+
+	/**
+	 * Constant holding the transient key for delayed triggered flush from activation.
+	 *
+	 * If this value is updated make sure you look at the method in the main class of TEC.
+	 *
+	 * @see TEC::activate
+	 *
+	 * @since 5.0.0.1
+	 *
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 903c54aea53c8f14934cea7e0c0f0229</sub>